decidim-proposals/app/forms/decidim/proposals/admin/proposal_answer_form.rb

```diff
@@ -18,12 +18,6 @@ class ProposalAnswerForm < Decidim::Form
         validates :internal_state, presence: true, inclusion: { in: :proposal_states }
         validates :answer, translatable_presence: true, if: ->(form) { form.state == "rejected" }
 
-        with_options if: :costs_required? do
-          validates :cost, numericality: true, presence: true
-          validates :cost_report, translatable_presence: true
-          validates :execution_period, translatable_presence: true
-        end
-
         alias state internal_state
 
         def costs_required?
```

Comment:
This method is no longer needed
The costs_enabled_method neither (line 37)

Reply:
These are required as functionality isn't present when filling the form.

Reply:
Where they are required? Can you point me to the actual line that requires them? Thanks

Reply:
Line 91 ```decidim-proposals/app/views/decidim/proposals/proposals/show.html.erb```

Reply:
@andreslucena let me revisit this because the functionality has changed since removing and reverting the helper: https://github.com/decidim/decidim/pull/14136#discussion_r1981201871